<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lobby</class>
 <widget class="QWidget" name="Lobby">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Public Room Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="general">
         <property name="title">
          <string>Nickname</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLineEdit" name="nickname">
            <property name="placeholderText">
             <string>Nickname</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filters">
         <property name="title">
          <string>Filters</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLineEdit" name="search">
            <property name="placeholderText">
             <string>Search</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="games_owned">
            <property name="text">
             <string>Games I Own</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="hide_full">
            <property name="text">
             <string>Hide Full Games</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="refresh_list">
           <property name="text">
            <string>Refresh Lobby</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="chat">
           <property name="text">
            <string>Chat</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="room_list"/>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
